### PR TITLE
Issue/15 init client app

### PR DESCRIPTION
**Pull request by papiliond**

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (barthamark) [inline client/src/index.css:1]: You can delete this and the _app.css_ files as we'll implement all the custom styles in the `useStyles` hook for the `MantineProvider`.

**Review by barthamark — CHANGES_REQUESTED** [review]
Please fix the linter error as well.

File: client/src/index.css (hunk 1 above)
Comment (papiliond) [reply client/src/index.css:1]: @barthamark I didn't know about this way. Let's use it 👍

Comment (papiliond) [reply client/src/index.css]: Fixed lint errors.

Comment (barthamark) [reply client/src/index.css]: @papiliond there are still linter errors for a few items.

Comment (papiliond) [reply client/src/index.css]: @barthamark For me both eslint and prettier gives successful output.

**Review by barthamark — APPROVED** [review]

**Pull request merged**